### PR TITLE
More Unit tests

### PR DESCRIPTION
**Pull request by janid1967**
Found couple of BUGS while writing tests
* RequirementDefinitionValidator.IsVoided checked if parent RequirementType was voided
* StepValidator.IsVoided checked if parent Journey was voided
* GetAllResponsiblesQueryHandler.cs was not implemented. Hard coded return of mocked data

### CODE REVIEW

**Comment by janid1967** [comment]
@hbjorgo Review of PR #46 is more important. Need that one to continue on the next command: PreserveTag

File: src/Equinor.Procosys.Preservation.Query/ResponsibleAggregate/GetAllResponsiblesQuery.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 namespace Equinor.Procosys.Preservation.Query.ResponsibleAggregate
 {
-    public class AllResponsiblesQuery : IRequest<Result<List<ResponsibleDto>>>
+    public class GetAllResponsiblesQuery : IRequest<Result<IEnumerable<ResponsibleDto>>>
```
Comment (hbjorgo) [inline src/Equinor.Procosys.Preservation.Query/ResponsibleAggregate/GetAllResponsiblesQuery.cs]: Return Collection<T> if possible. Ref. guidelines:

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections#collection-parameters

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections#collection-properties-and-return-values

File: src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/JourneyValidatorTests.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        private const int JourneyIdNonVoided = 1;
+        private const int JourneyIdVoided = 2;
+        private JourneyValidator _dut;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            var journeyRepositoryMock = new Mock<IJourneyRepository>();
+
+            var journey = new Journey("S", "J");
```
Comment (hbjorgo) [inline src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/JourneyValidatorTests.cs]: Use constant for JourneyTitle.

File: src/Equinor.Procosys.Preservation.Query/ResponsibleAggregate/GetAllResponsiblesQuery.cs (hunk 1 above)
Comment (janid1967) [reply src/Equinor.Procosys.Preservation.Query/ResponsibleAggregate/GetAllResponsiblesQuery.cs]: U fix

File: src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/ModeValidatorTests.cs [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+using Equinor.Procosys.Preservation.Domain.AggregateModels.ResponsibleAggregate;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Equinor.Procosys.Preservation.Command.Tests.Validators
+{
+    [TestClass]
+    public class ModeValidatorTests
+    {
+        private const string ModeTitle = "J";
```
Comment (hbjorgo) [inline src/tests/Equinor.Procosys.Preservation.Command.Tests/Validators/ModeValidatorTests.cs]: Could this be used when mocking modes?

**Pull request merged**